recipes-connectivity/pe-terminal/pe-terminal_0.0.1.bb

```diff
@@ -20,7 +20,7 @@ file://${BPN}-watcher.path \
 
```

Comment:
I would change the filename to be `_1.0.1.bb`, as we're not anymore on version 0.0.1.
There's a tag already in the pe-terminal repo with 1.0.0.

Reply:
Simply renaming the file doesn't change what gets taken from git. For that we need to change the SRCREV to include the version of the recipe, and have pe-terminal tagged accordingly.